File: QTD2.Domain/Entities/Core/EmployeeOrganization.cs
namespace QTD2.Domain.Entities.Core
{
    public class EmployeeOrganization : Common.Entity
    {
        public int EmployeeId { get; set; }

        public int OrganizationId { get; set; }

        public bool IsManager { get; set; }

        public virtual Employee Employee { get; set; }

        public virtual Organization Organization { get; set; }

        public EmployeeOrganization(int employeeId, int organizationId, bool isManager)
        {
            EmployeeId = employeeId;
            OrganizationId = organizationId;
            IsManager = isManager;
        }

        public EmployeeOrganization()
        {
        }
    }
}

File: QTD.SPA/src/app/_DtoModels/EmployeeOrganization/EmployeeOrganization.ts
import { Employee } from '../Employee/Employee';
import { Entity } from '../Entity';
import { Organization } from '../Organization/Organization';

export class EmployeeOrganization extends Entity {
  employeeId!: any;
  organizationId!: any;
  employee!: Employee;
  organization!: Organization;
  isManager : boolean;
  firstName:any;
  lastName:any;
}
